internal/node/node.bzl

```diff
@@ -209,12 +223,66 @@ _NODEJS_EXECUTABLE_ATTRS = {
         allow_files = True,
         aspects = [sources_aspect, module_mappings_runtime_aspect, collect_node_modules_aspect],
     ),
-    "entry_point": attr.string(
+    "entry_point": attr.label(
         doc = """The script which should be executed first, usually containing a main function.
-        This attribute expects a string starting with the workspace name, so that it's not ambiguous
-        in cases where a script with the same name appears in another directory or external workspace.
+        The `entry_point` accepts a target's name as an entry point. 
```

Comment:
"target" is confusing here, ppl don't think of their input .js files as being targets

Reply:
removed that line